src/java/com/zimbra/cs/taglib/tag/i18n/I18nUtil.java

```diff
@@ -499,6 +501,45 @@ String setSkin(ServletRequest request, ServletResponse response) {
     //
     // ClassLoader methods
     //
+    @Override
+    public URL getResource(String filename) {
+        if (ZimbraLog.webclient.isDebugEnabled()) {
+            ZimbraLog.webclient.debug("getResource: filename="+filename);
```

Comment:
Would generally prefer the style of logging `ZimbraLog.webclient.debug("getResource: filename=%s", filename)`.  If you used that, there would be less need to guard logging with `isDebugEnabled()` etc as would no longer be creating useless objects when not logging.